src/views/Viewer.vue

```diff
@@ -900,6 +906,17 @@ export default {
 		}
 	}
 
+	&.theme--dark::v-deep .button-vue--vue-tertiary {
```

Comment:
We should be able to enforce the theme used more beautifully with css variables now
As well with the data-theme attribute @juliushaertl